dpdispatcher/ssh_context.py

```diff
@@ -648,26 +648,27 @@ def download(
         # for ii in job_dirs :
         for task in submission.belonging_tasks:
             for jj in task.backward_files:
-                file_name = pathlib.PurePath(
-                    os.path.join(task.task_work_path, jj)
-                ).as_posix()
+                file_name_list = glob(
```

Comment:
It isn't very clear how `glob` works for a remote path.

Reply:
Thanks, I think it can work now.
And the bug of using`glob` to get error* files is fixed now.